backend/curriculum_tracking/serializers.py

```diff
@@ -611,3 +611,21 @@ def get_oldest_card_in_review_time(self, instance):
 
     def get_total_cards_in_review(self, instance):
         return self._get_review_agile_cards(instance).count()
+
+
+class ReviewTrustSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = models.ReviewTrust
+        fields = [
+            "id",
+            "content_item",
+            "flavours",
+            "user",
+        ]
+
+    content_item = serializers.SerializerMethodField('get_name_not_id_of_content_item')
+    flavours = serializers.CharField(help_text="comma separated list of flavours")
+
+    def get_name_not_id_of_content_item(self, review_trust: object):
+        content_item_name = review_trust
+        return str(content_item_name)
```

Comment:
This seems a little bit strange. Firstly, there is an extra line of code:
```
def get_name_not_id_of_content_item(self, review_trust: object):
        content_item_name = review_trust
        return str(content_item_name) 
```

This does the same thing:

```
def get_name_not_id_of_content_item(self, review_trust: object):
        return str(review_trust) 
```

But then you are relying on the str method of the review trust, so if anyone wants to see how this actually works they have to go on a treasure hunt. Also, what if someone changes how ReviewTrust.__str__ works? Then your api endpoint would randomly change and then the frontend would randomly act differently. 

It's best to be explicit when dealing with data.

```
def get_content_item_title(self, review_trust):
    return review_trust.content_item.title 
```